cloud/scope/powervs_cluster_test.go

```diff
@@ -3830,3 +3832,799 @@ func TestCreateVPCSubnet(t *testing.T) {
 		g.Expect(err).ToNot(BeNil())
 	})
 }
+
+func TestIsResourceCreatedByController(t *testing.T) {
+	testCases := []struct {
+		name           string
+		resourceType   infrav1beta2.ResourceType
+		clusterScope   PowerVSClusterScope
+		expectedResult bool
+	}{
+		{
+			name: "When resourceType is VPC and VPC status is nil",
+			clusterScope: PowerVSClusterScope{
+				IBMPowerVSCluster: &infrav1beta2.IBMPowerVSCluster{},
+			},
+			resourceType:   infrav1beta2.ResourceTypeVPC,
+			expectedResult: false,
```

Comment:
bool will be default be false. so we can remove it or keep it for better readability

Reply:
I have kept it as is for easy readability